### PR TITLE
CI: update CI tests, support vulkan backend

### PR DESCRIPTION
**Pull request by jmoguillansky-gpsw**

### CODE REVIEW

File: .github/workflows/ci_linux.yml [hunk 1]
```diff
@@ -10,6 +10,8 @@ jobs:
   linux:
 
     runs-on: ubuntu-20.04
+    env:
+        VULKAN_SDK_VERSION: 1.2.170
```
Comment (mbouron-gpsw) [inline .github/workflows/ci_linux.yml:14]: Can't we just depend on mesa ? I guess ninja-build renderdoc is useful for master, can you do a separate PR with this two dependencies targetting master ?

Comment (jmoguillansky-gpsw) [reply .github/workflows/ci_linux.yml:14]: The goal is to confirm that vulkan backend builds ok in ci test.

Comment (mbouron-gpsw) [reply .github/workflows/ci_linux.yml:14]: I mean, can we rely on system packages ? like libvulkan-dev ?

Comment (mbouron-gpsw) [reply .github/workflows/ci_linux.yml:14]: Looks like shaderc is not packaged on Ubuntu so the commit is fine.
